index.d.ts

```diff
@@ -130,6 +156,8 @@ export class Notification {
    * This Object is JSON encoded and sent as the notification payload. When properties have been set on notification.aps (either directly or with convenience setters) these are added to the payload just before it is sent. If payload already contains an aps property it is replaced.
    */
   public payload: any;
+  public aps: Aps;
```

Comment:
I am not sure about exposing the `aps` property. If you want to modify the raw payload, you should use the `rawPayload` property. But I am open for discussions.